### ISSUE TITLE
One more asterisk is added when start a new line from the end of the block

### ISSUE THREAD

**Issue by zp-j** [nikhilkalige/docblockr#131]
```javascript
/**************************
 * Hello world            *
 * Bla bla bla bla bla    *
 **************************/ (enter here)
```

When I start a new line from the end of the block, one more asterisk will be added:

```javascript
/**************************
 * Hello world            *
 * Bla bla bla bla bla    *
 **************************/
 *
```

**Comment by zp-j** [comment]
Hi @nikhilkalige @malko, I just added a small check to the end of the regular expression: [https://github.com/zp-j/docblockr/blob/2c4717a45c70cc725cc42b1e4939a053a5207914/lib/docblockr-worker.js#L69](https://github.com/zp-j/docblockr/blob/2c4717a45c70cc725cc42b1e4939a053a5207914/lib/docblockr-worker.js#L69), can you please review that? Thanks!

**Comment by MoritzKn** [comment]
This was fixed by #131. In version v0.8.1 is shouldn't happen anymore.
